src/main/java/uk/ac/ebi/eva/pipeline/Application.java

```diff
@@ -34,6 +34,7 @@
 public class Application {
 
     private static final Logger log = LoggerFactory.getLogger(Application.class);
+    public static final String VARIANT_ANNOTATION_MONGO_PROFILE = "variant-annotation-mongo";
```

Comment:
We could use a more generic MongoDB profile, which would allow to easily configure **all** the readers and writers in the pipeline when changing a back-end, as well as simplify the .properties definition.

On the other hand, we wouldn't be able to test reading from an existing Mongo and writing into another technology under testing. But this may be not a use case to really consider. What do you think @jorizci ?